app/api/models/statistics.py

```diff
@@ -7,9 +7,21 @@
 from loguru import logger
 
 class Homology(BaseModel):
-    coverage: float = Field(alias="homology_coverage", default=None)
-    coverage_explanation: str = None
-
+    coverage: float = Field(alias="compara.homology_coverage ", default=None)
+    reference_species_name: str = Field(alias="compara.homology_reference_species ", default=None)
+
+    @validator('reference_species_name', pre=True)
+    def validate_na_values(cls, v ) -> str:
+        rsn = ""
```

Comment:
What is rsn? Can we please use full names like we do everywhere in the code?

Reply:
Done